src/scss/_sofie-forms.scss

```diff
@@ -72,3 +71,12 @@ select {
     right: 10px;
   }
 }
+
+.c-multi-select-menu {
```

Comment:
Why is this a global CSS class instead of being encapsulated in the MultiSelect component?